conda/recipes/libcudf/meta.yaml

```diff
@@ -33,14 +33,16 @@ build:
 
 requirements:
   build:
-    - cmake >=3.17.0
+    - cmake >=3.18
   host:
     - librmm {{ minor_version }}.*
     - cudatoolkit {{ cuda_version }}.*
     - arrow-cpp 1.0.1
     - arrow-cpp-proc * cuda
     - boost-cpp 1.72.0
     - dlpack
+    - gtest
+    - gmock
```

Comment:
Given these are only required if someone uses `cudftestutil`, how would we feel about not including these and if someone uses `cudftestutil` they're required to have `gtest` / `gmock` installed in their environment?

Reply:
That's fine with me, just means we have to install them into the gpuCI containers and block merging until the next nightly release.

Reply:
this change was in reference to https://github.com/rapidsai/cuspatial/pull/365#discussion_r601686649

Reply:
They should already be in the gpuci build environment: https://github.com/rapidsai/integration/blob/branch-0.19/conda/recipes/rapids-build-env/meta.yaml#L78-L80

Reply:
Have we had a run on gpuCI where these didn't work? We install the `rapids-build-env` in all gpuCI images that we build with so both `gtest`/`gmock` should be there

Reply:
[This cuSpatial build](https://gpuci.gpuopenanalytics.com/job/rapidsai/job/gpuci/job/cuspatial/job/prb/job/cuspatial-cpu-cuda-build/CUDA=11.0/50/consoleText) failed because CMake couldn't find GTest:

>   cudf could not be found because dependency GTest could not be found.

But the real reason could due to cuDF not having the changes in this PR. I'll remove these here and cuSpatial and see what happens.